diff --git a/node_modules/@idleflowgames/capacitor-play-games/android/build.gradle b/node_modules/@idleflowgames/capacitor-play-games/android/build.gradle
index ea93fb5..23347a5 100644
--- a/node_modules/@idleflowgames/capacitor-play-games/android/build.gradle
+++ b/node_modules/@idleflowgames/capacitor-play-games/android/build.gradle
@@ -22,6 +22,7 @@ buildscript {
 }
 
 apply plugin: 'com.android.library'
+apply plugin: 'org.jetbrains.kotlin.android'
 
 android {
     namespace = "com.idleflowgames.playgames"
